src/test/java/com/epam/tc/hw2/ex2/TestExercise2.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.tc.hw2.ex2;
+
+import com.epam.tc.hw2.WebDriverSetup;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class TestExercise2 extends WebDriverSetup {
+    @Test
+    public void checkExercice2() {
```

Comment:
please, use description parameter of annotation to define what exercise the task is devoted to